client/my-sites/subscribers/main.tsx

```diff
@@ -80,23 +80,24 @@ const SubscribersPage = ( { pageNumber, pageChanged }: SubscribersProps ) => {
 	return (
 		<Main wideLayout className="subscribers">
 			<DocumentHead title={ translate( 'Subscribers' ) } />
-			<FixedNavigationHeader navigationItems={ navigationItems }>
-				<Button
-					className="add-subscribers-button"
-					primary
-					onClick={ () => setShowAddSubscribersModal( true ) }
-				>
-					<Gridicon icon="plus" size={ 24 } />
-					{ translate( 'Add subscribers' ) }
-				</Button>
-				<SubscribersHeaderPopover siteId={ selectedSiteId } />
-			</FixedNavigationHeader>
 
 			<SubscribersListManagerProvider
 				siteId={ selectedSiteId }
 				page={ pageNumber }
 				pageChanged={ pageChanged }
 			>
+				<FixedNavigationHeader navigationItems={ navigationItems }>
+					<Button
+						className="add-subscribers-button"
+						primary
+						onClick={ () => setShowAddSubscribersModal( true ) }
+					>
+						<Gridicon icon="plus" size={ 24 } />
+						{ translate( 'Add subscribers' ) }
+					</Button>
+					<SubscribersHeaderPopover siteId={ selectedSiteId } />
+				</FixedNavigationHeader>
+
```

Comment:
Since `grandTotal` is required to conditionally hide / display `SubscribersHeaderPopover`, I have wrapped the  `FixedNavigationHeader ` inside the `SubscribersListManagerProvider` - as it provides context that includes the `grandTotal` value.

What do you think about this approach?

I considered lifting the `grandTotal` up, but that felt like too much work just to get to this one value. Also, I didn't feel it was right to wrap the whole `SubscribersPage` in the `SubscribersListManagerProvider`.

Another option I considered was to find a different value we could use to conditionally display / hide the `SubscribersHeaderPopover` (instead of `grandTotal`), but I didn't find anything useful.

Reply:
I think it is OK to move fixed navigation header out out as a separate component, maybe call it "SubscribersHeader", inside that you can use the context to hide or show the popover.

And then have the whole `<Main>` wrapped in the `<SubscribersListProvider>`.

Reply:
Thank you, Yan!

> And then have the whole `<Main>` wrapped in the `<SubscribersListProvider>`.

I wonder if this isn’t in the conflict with the [readme instructions](https://github.com/Automattic/wp-calypso/tree/trunk/client/my-sites) for my-sites sections where it states that the outer component should be `<Main>`. 🤔 

> The first component that is rendered for any section in #primary should be called section/main.jsx. That component should make use of components/main for it's wrapper.

Reply:
It should be 100% okay to move that in. Might be an idea to rename `SubscribersListManagerProvider` to `SubscribersPageProvider` or something like that.

Reply:
Thank you, Tim!

Reply:
I have applied all the suggestions.